tests/conflicts.rs

```diff
@@ -2,20 +2,27 @@ mod utils;
 
 use clap::{App, Arg, ArgGroup, ErrorKind};
 
-static CONFLICT_ERR: &str = "error: The argument '-F' cannot be used with '--flag'
+static CONFLICT_ERR: &str = "error: The argument '--flag' cannot be used with '-F'
 
 USAGE:
     clap-test <positional> <positional2> --flag --long-option-2 <option2>
 
 For more information try --help";
 
-static CONFLICT_ERR_REV: &str = "error: The argument '--flag' cannot be used with '-F'
+static CONFLICT_ERR_REV: &str = "error: The argument '-F' cannot be used with '--flag'
```

Comment:
The original messages are correct. They shouldn't be swapped.

Reply:
I'd say this isn't very important as long as it's deterministic.

Reply:
Yes, but it should be easily fixable, so, why not try it?

Reply:
I don't believe the original messages are correct. Or more specifically, I believe either these messages are incorrect, or the assertions in `two_conflicting_arguments` and `three_conflicting_arguments` are incorrect.

From the PR description:
> I edited the `CONFLICT_ERR` and `CONFLICT_ERR_REV` static strings because they did not follow the convention that other tests were following, i.e. mention the conflicting parameters in the order in which they appear (e.g. `two_conflicting_arguments` or `three_conflicting_arguments`).

I'll expand on that here to be more clear. Before my change this was state of both some existing tests and the ones using these strings:
```
+-----------------------------+----------------------+----------------------------+
| Test                        | input argument order | test assertion order       |
+-----------------------------+----------------------+----------------------------+
+-----------------------------+----------------------+----------------------------+
| two_conflicting_arguments   | develop, production  | develop, production        |
+-----------------------------+----------------------+----------------------------+
| three_conflicting_arguments | one, two, three      | one, two                   |
+-----------------------------+----------------------+----------------------------+
| conflict_output             | flag, F              | F, flag (CONFLICT_ERR)     |
+-----------------------------+----------------------+----------------------------+
| conflict_output_rev         | F, flag              | flag, F (CONFLICT_ERR_REV) |
+-----------------------------+----------------------+----------------------------+
```
It looks like it is desired that the conflicting arguments are listed in the order they appeared in the input, so I changed the strings to match that understanding. I'm happy to change it back, but if I do, then shouldn't the assertions in the other tests change? If neither, how are we supposed to choose the order in which to display the conflicting arguments in the message?

Reply:
> It looks like it is desired that the conflicting arguments are listed in the order they appeared in the input

I say again it's not a big deal, but yes, the "like in the input" order is preferable. I think you've done right.

> how are we supposed to choose the order in which to display the conflicting arguments in the message?

Theoretically speaking, "lexicographically sorted" might be an alternative order. But I think your order is better.

Reply:
Let's say we have `clap --flag -F` given as input:

The error message is `The argument '--flag' cannot be used with '-F'` and the usage is `clap --flag`

The usage is good and correct here since it uses the input order. But, if you look at the error message, the way it describes is that `-F` already exists and then the user provided `--flag` which is the reverse of what happened here. If the error message was phrased differently then the order you had would be okay.

It's upto you if you want to change the message phrasing itself. I just don't want to confuse the users and try to be as clear as possible.

If you decide to change the order, please do for all the test cases.

Reply:
Understood. I see how the language could be confusing. I have changed the message to the reverse input order, reverted the messages back to the original, and adjusted the other test assertions to match.